src/test/java/org/jenkinsci/plugins/pipeline/utility/steps/conf/WriteYamlStepUnitTest.java

```diff
@@ -20,11 +21,26 @@ public void writeEmptyListObject() throws Exception {
         WriteYamlStep writeStep = new WriteYamlStep("/dev/null", new ArrayList<>());
     }
 
+    @Test @WithoutJenkins
+    public void writeEmptyListOfDocuments() throws Exception {
```

Comment:
What is this testing? It doesn't look like it is testing anything at all.
Or do you mean to test that it doesn't throw an exception when you specify an empty list?

Reply:
It's meant to be an analogue of `writeEmptyListObject`, which is to test that it doesn't throw an exception on an empty list. Just this one passes the empty list as datas instead of data.